pom.xml

```diff
@@ -1305,6 +1305,9 @@
           </plugin>
         </plugins>
       </build>
+      <properties>
```

Comment:
As noted in https://github.com/jenkinsci/pom/pull/27#pullrequestreview-137653976 this would better be done using mojo configuration.